components/org.wso2.carbon.database.utils/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
ADd the license header

Reply:
Commit message should be in imperative form. e,g Add

Reply:
Done